testsuite/pom.xml

```diff
@@ -28,12 +28,16 @@
         <version.wildfly10>10.0.0.Final</version.wildfly10>
         <version.wildfly10.core>2.0.10.Final</version.wildfly10.core>
         <version.wildfly10.arquillian>1.1.0.Alpha1</version.wildfly10.arquillian>
+        <version.wildfly11>11.0.0.Alpha1-SNAPSHOT</version.wildfly11>
```

Comment:
Hmm, OK, but at least add a comment. Something like:

```xml
<!-- doesn't exist yet; update when there's a public release of WildFly 11 -->
<version.wildfly11>11.0.0.Alpha1</version.wildfly11>
```

And a similar comment to the `wildfly11` profile.